nx-dev/nx-dev/redirect-rules.config.js

```diff
@@ -174,7 +174,7 @@ const overviewUrls = {
   '/next/overview': '/packages/next',
   '/detox/overview': '/packages/detox',
   '/react-native/overview': '/packages/react-native',
-  '/packages/nx-plugin': '/packages/nx-plugin',
```

Comment:
I wonder if it would be worth writing some kind of test that asserts that the redirect doesn't point to itself? I recognize that this is a typo and probably wouldn't happen again, but the test would be pretty easy if we made a redirect-rules.config.spec.ts